earth2mip/inference_ensemble.py

```diff
@@ -258,7 +260,19 @@ def perturb(x, rank, batch_id, device):
             )
         if rank == 0 and batch_id == 0:  # first ens-member is deterministic
             noise[0, :, :, :, :] = 0
+        center = torch.tensor(
+            [channel_means[channel] for channel in model.in_channel_names],
```

Comment:
this will break the code, I can add a check. but I took the largest channel set we have (235 channels) at the moment all models use a subset of it

Reply:
@NickGeneva  an alternative might be to require that each model that we bring it will make sure they have the mean and stds of the channels the model use, not sure how or where to place these ? 
@nbren12 , wouldn't this break with Graphcast given the channel of incoming solar radiation? maybe the idea of computing the mean and std of the initial state and use them would be a better approach after all ?

Reply:
I am fine with no check here. We can add new channels  as needed to the list, this one is pretty comprehensive. 

TISR is not a blocker. It is a forcing and not in `in_channel_names`. It should not be perturbed.

Reply:
> I can add a check. 

please no check. The code will give a nice KeyError all on its own. no need to jump the gun.

Reply:
I alway prefer

```
d[key]
```

to 

```
if key in d:
    d[key]
else:
    raise KeyError
```

Reply:
@NickGeneva I've seen many bugs related to such "helpful" logic. Sometimes the error isn't actually an error. for example, if `d` did not having `__contains__` but did have `__getitem__`. 

If you want to layer more helpful info (in this case the error message is almost identical), I think this construction should be preferred:

```
try:
    d[key]
except KeyError:
   raise KeyError("your potentially more helpful error message")